packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -322,6 +323,13 @@ export class MirrorNodeClient {
             requestId);
     }
 
+    public async getAccountPageLimit(idOrAliasOrEvmAddress: string, requestId?: string) {
+        return this.get(`${MirrorNodeClient.GET_ACCOUNTS_ENDPOINT}${idOrAliasOrEvmAddress}?limit=${MirrorNodeClient.PAGE_LIMIT}`,
```

Comment:
This already exists with `constants.MIRROR_NODE_QUERY_LIMIT`
```suggestion
        return this.get(`${MirrorNodeClient.GET_ACCOUNTS_ENDPOINT}${idOrAliasOrEvmAddress}?limit=${constants.MIRROR_NODE_QUERY_LIMIT}`,
```

Reply:
Updated